UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Extensions/LoggerExtensions.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using UKHO.ExchangeSetService.Common.Logging;
+
+namespace UKHO.ExchangeSetService.Common.Extensions
+{
+    public static class LoggerExtensions
+    {
+        public static void LogStartEndAndElapsedTime<T>(this ILogger<T> logger,
+            EventIds startEventId,
+            EventIds completedEventId,
+            string messageFormat,
+            Action action,
+            params object[] messageArguments)
+        {
+            logger.LogStartEndAndElapsedTime(startEventId, completedEventId, messageFormat, () =>
+            {
+                action();
+                return 1;
+            }, messageArguments);
+        }
+
```

Comment:
This is unused and can be removed please
```suggestion
```

Reply:
Yes. removed method.